### PR TITLE
Encrypt session token

### LINKED ISSUE #432: Encrypt access token in session
Currently it's only done for those in checks.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #432

### CODE REVIEW

**Comment by prayerslayer** [comment]
👍

**Comment by mfellner** [comment]
@prayerslayer Please change the commit messages to match the Zappr pattern! 😿

**Comment by prayerslayer** [comment]
Done.

File: server/model/Database.js [hunk 1]
```diff
@@ -11,16 +12,56 @@ const encryptionService = EncryptionServiceCreator.create()
...
+   */
+  if (thing) {
+    /**
+     * Repository with list of checks (that have tokens)
+     */
+    if (Array.isArray(thing.checks)) {
+      log('decrypt token hook')
+      await Promise.all(thing.checks.map(async(c) => {
+        const plain = await encryptionService.decrypt(c.token)
+        if (typeof thing.set === 'function') {
```
Comment (mfellner) [inline server/model/Database.js]: This check should be part of the first "if"

**Comment by mfellner** [comment]
👍

**Comment by prayerslayer** [comment]
👍

**Pull request merged**